src/main/java/EigenValue.java

```diff
@@ -0,0 +1,59 @@
+public class EigenValue {
+
+    public static void main(String[] args) {
+        int[][] matrix = new int[][]{{2, -1}, {-1, 2}};
+        getEigenValues(matrix);
+    }
+
+    public static void getEigenValues(int[][] matrix) {
```

Comment:
get* are getter methods by the default style, that solves accessibility / visibility problems and has very low resource usage. Rename the method